cypress/integration/Modal.spec.js

```diff
@@ -2,18 +2,36 @@
 
 describe('The Modal component', () => {
   beforeEach(() => {
```

Comment:
Should we add a test here to check `focused` is within the modal? Also wondering if it makes sense to render the close button after modal `children` – it seems its will be the first element focused

Reply:
Nevermind, didn't see the test below that tabs through